app/views/court_dates/_form.html.erb

```diff
@@ -13,18 +13,14 @@
 
       <div class="field form-group">
         <%= form.label :date, "Add Court Date" %>
-        <br>
-        <span class="datetime-year-month">
-          <%= form.date_select :date,
-                                {
-                                    order: [:day, :month, :year],
-                                    start_year: Date.current.year + 3,
-                                    end_year: 2000,
-                                    prompt: {day: t("common.day"), month: t("common.month"), year: t("common.year")}
-                                },
-                                class: "select2 date-input" %>
-        </span>
+         <% court_date = casa_case.court_date || Time.zone.now %>
```

Comment:
Now might be a little weird here, 6 months in the future might be better? But now is "normal" UI

Reply:
no idea. I did what the example also did.
Can take a look if I can change it  to be now and 6 months later ?